ci-operator/jobs/openshift/installer/openshift-installer-main-presubmits.yaml

```diff
@@ -6125,6 +6125,81 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Let's make running this conditional, perhaps `always_run: false` && `run_if_changed: vsphere`?